test/runtests.jl

```diff
@@ -121,6 +121,7 @@ Si        13.00000000      14.00000000      $(frame+1).00000000          0
             ExtXYZ.save(outfile, seq1)
             seq2 = ExtXYZ.load(outfile)
             @test all(seq1 .≈ seq2)
+            @test all(repr(seq1) == repr(seq2))
```

Comment:
I had more in mind to do something like `@test eval(Meta.parse(repo(seq1))) ≈ seq1`, but this may not work with the output formats used in AtomsBase

Reply:
I just tried it and it seems not to work, in the case for your testfile it says 

> Si not defined

I think the atomic_symbols are a problem here.

Reply:
OK, that's fine then.